parameters/parameters.go

```diff
@@ -4,6 +4,7 @@
 package parameters
 
 import (
+	"github.com/santhosh-tekuri/jsonschema/v6"
```

Comment:
The import order should be fixed.

Use goimports

Reply:
Ack ... to much reliance on GoLand ;-)